content/en/operator-registry/_index.md

```diff
@@ -1,8 +1,7 @@
 
 ---
 title: "Operator Registry Documentation"
-linkTitle: "Registry Documentation"
-weight: 20
+linkTitle: "Operator-registry"
```

Comment:
@njhale that does make sense. How do we feel about having this under `Concepts` -> `Operator-Registry`

Reply:
We can probably address this in a followup.

> How do we feel about having this under Concepts -> Operator-Registry

Depending on the goals of this doc site, it may make sense to spread the content across `docs` instead of concentrating it in a repo-specific top-level directory.